aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,25 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## February 14 - March 4, 2022
+
+### Communications
+- **Office Hours** - We have combined our two office hours into one which will be held every Thursday, 10:00 PM IST / 8:00 AM PST. This office hours is open to all. Contact Kamlesh Thakur, if you would like a calendar invite.
+
+### Roadmap Updates 
+- **Page Layouts** - Completed dashboard template for a [3 column dashboard layout](https://design-system.hpe.design/templates/dashboards#three-column-dashboard).
+- **Page Header** - Completed research and use case gathering for Page Header and began design ideation.
+- **Banner Notification** - Completed ideation on Banner Notifications. Guidance, theme updates, and implementation examples for Banner Notifications will be coming soon.
```

Comment:
```suggestion
- **Banner Notification** 
  - Completed ideation on Global Banner Notifications.
  - Added support for [Global Banners and actions](https://github.com/grommet/grommet/pull/5958) to Grommet's Notification component.
  - Guidance, theme updates, and implementation examples for Global Banner Notifications are coming soon.
```

Reply:
so I was going back and forth on adding this information referring to the Notification that is on the Grommet stable release just since we haven't added guidance and wasn't sure if users may use it the incorrect way